tools/js-sdk-release-tools/src/autoGenerateInPipeline.ts

```diff
@@ -65,7 +65,7 @@ async function automationGenerateInPipeline(inputJsonPath: string, outputJsonPat
             cadlProject: cadlProject,
             autorestConfig,
             use: use,
-            cadlEmitter: !!cadlEmitter? cadlEmitter : `@azure-tools/cadl-typescript`,
+            cadlEmitter: !!cadlEmitter? cadlEmitter : `@azure-tools/typespec-ts`,
```

Comment:
Shall we still use 'cadlEmitter'?

Reply:
It would be good to do a full search for all cadl in the source code.

Reply:
Sure, let me update the name accordingly